components/eamxx/src/control/intensive_observation_period.cpp

```diff
@@ -502,8 +503,10 @@ read_fields_from_file_for_iop (const std::string& file_name,
     // Sync fields to device
     fm_field.sync_to_dev();
 
-    // Set the initial time stamp on FM fields
-    fm_field.get_header().get_tracking().update_time_stamp(initial_ts);
+    // Set the initial time stamp on FM fields (if a valid TimeStamp is given)
+    if (initial_ts.is_valid()) {
```

Comment:
Why would initial_ts not be valid?

Reply:
Good point, there is not reason we can't set the timestamp for SPA fields (it just wasn't being done). I'll remove this check and pass the current timestamp.